### ISSUE TITLE
`_Complex` keyword is unsupported

### ISSUE THREAD

**Issue by andrewvaughanj** [ltcmelo/psychec#10]
**If eligible for the bounty, I wish this to be donated to a local animal charity of the maintainer's choice**

#### GCC version:
```bash
gcc (SUSE Linux) 10.2.1 20201202 [revision e563687cf9d3d1278f45aaebd03e0f66531076c9]
```

#### clang version:
```bash
clang version 11.0.1
```

#### Output of `gcc -Werror -Wall -Wextra -Wpedantic -pedantic -std=c11 -c $FNAME`
```bash
(return code 0)
```

#### Output of `clang -Werror -Wall -Wextra -Wpedantic -pedantic -std=c11 -c $FNAME`
```bash
(return code 0)
```

#### Output of `cnip $FNAME`
```bash
pr90840.c:1:14 warning: missing type specifier, assume `int'
# 1 "<stdin>"
              ^
pr90840.c:2:9 error: expected , or `;' or = after declarator, got `double'
_Complex double qux_c;
         ^

(return code 0)
```

#### Content of file
```bash
_Complex double qux_c;
```

#### `psychec` version
```bash
commit 6d7b8249e0f28af105112b217cb33a24f666e148
Author: Leandro T. C. Melo <LTCMELO@GMAIL.COM>
Date:   Tue Feb 23 07:47:25 2021 -0300

    link expression-parsing table in Doxygen documentation
```

**Comment by ltcmelo** [comment]
Thank you for the report @andrewvaughanj !

**Comment by ltcmelo** [comment]
Fixed with 3e756d663033c0e72c9421987ff82106b44cfbd6.
